openebs/openebs-provisioner.go

```diff
@@ -51,6 +51,7 @@ type openEBSProvisioner struct {
 
 // NewOpenEBSProvisioner creates a new openebs provisioner
 func NewOpenEBSProvisioner(client kubernetes.Interface) controller.Provisioner {
+
 	nodeName := os.Getenv("NODE_NAME")
 	if nodeName == "" {
 		glog.Errorf("env variable NODE_NAME must be set so that this provisioner can identify itself")
```

Comment:
The error message need not be a detailed description. It can be: ENV variable 'NODE_NAME' is not set
Check if there are any best practices for logging if you can.